sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -108,9 +111,16 @@ struct {
 }
 ```
 </td>
-</tr><tr>
-<td rowspan="2">queue</td>
-<td rowspan="2"><pre>ze_command_queue_handle_t</pre></td>
+</tr>
+<tr>
+<td rowspan="3">queue</td>
+<td rowspan="2">
+
+``` C++
+ze_command_queue_handle_t
+```

```

Comment:
Add the same phrasing here in this column: "Prior to version 4 of this specification" and "Starting in version 4 of this specification".  Users need to know which definition is used in each version of the spec.

Reply:
I don't see this change yet (or the other one about "NOTE").  Maybe you forgot to push a commit?

Reply:
Sorry about that. Please try now.